taskvine/src/manager/vine_manager.c

```diff
@@ -2610,14 +2611,18 @@ struct rmsummary *vine_manager_choose_resources_for_task(struct vine_manager *q,
 	const struct rmsummary *min = vine_manager_task_resources_min(q, t);
 	const struct rmsummary *max = vine_manager_task_resources_max(q, t);
 
-	/* available disk for all sandboxes */
-	int64_t available_disk = w->resources->disk.total - BYTES_TO_MEGABYTES(w->inuse_cache);
+	/* total disk for all sandboxes */
+	int64_t disk_total = w->resources->disk.total - BYTES_TO_MEGABYTES(w->inuse_cache);
 
-	/* do not count the size of input files as available.
+	/* do not count the size of input files as the total.
 	 * TODO: efficiently discount the size of files already at worker. */
-	if (t->input_files_size > 0) {
```

Comment:
We need this `if`.  `input_files_size` is -1 when unset, so you do not want to -(-1) = +1 a MB to the available disk.